### DISCUSSION TITLE
you are swearing in the script

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1497] [Q&A]
## What is your question?
no swearingplz strike 1

**Comment by frosthead69** [comment]
what

**Comment by droni1234** [comment]
already heared about freedom of speech man???

**Comment by ghost** [comment]
no but have you heard of heck????

**Comment by actuallyasmartname** [comment]
There's literally offensive language in one of the splash texts lol https://github.com/ninjamuffin99/Funkin/blob/master/assets/preload/data/introText.txt

**Comment by ghost** [comment]
lol

**Comment by isakube** [comment]
*proceeds to rated T on Newgrounds*
